fluent-jdbc/src/main/java/org/codejargon/fluentjdbc/internal/mappers/ObjectMapper.java

```diff
@@ -94,7 +94,7 @@ private Object optionalOf(Field field, Object value) {
             return Optional.empty();
         } else {
             Class<?> typeOfField = typeOfField(field);
-            if (!typeOfField.isAssignableFrom(value.getClass())) {
+            if (!value.getClass().isAssignableFrom(typeOfField)) {
```

Comment:
this change was required for detecting a ByteBuffer instances like the package internal HeapByteBuffer or DirectByteBuffer. I will try to find a better way and for now please remove the whole ByteBuffer stuff. The byte[] still works without any dirty workaround. I'm a beginner in reading writing groovy and my next pull requests contains units test too :)